docs/src/pages/components/authenticator/LabelsAndTextDemo.tsx

```diff
@@ -71,15 +71,28 @@ function Screen({ Component }: ScreenProps) {
 }
 
 export function LabelsAndTextDemo({ Component }: ScreenProps) {
+  const OnMachineInit = ({ children }) => {
+    /**
+     * This waits for Authenticator machine to init before its inner components
+     * start consuming machine context.
+     */
+    const { route } = useAuthenticator();
+    if (!route || route === 'idle' || route === 'setup') return null;
+
+    return <>{children}</>;
+  };
```

Comment:
This is the only additional change since #1142.

Labels and Text demo was using `Authenticator.<SubComponent>` directly, which relied on `authMachine` to be already initialized and ready. With #1142, `authMachine` isn't fully initiated when this demo renders, which caused a runtime error. This component goes around it by rendering demo only if the machine is set up.

Reply:
https://user-images.githubusercontent.com/43682783/150034299-bbdc5c5e-178a-4c86-9349-163a3080a292.mp4

Reply:
Good enough for me!

Reply:
And there is no where else we use these subcomponents?

Reply:
Yep, this is the only place i could find it used 👍